nix/build/hacks.nix

```diff
@@ -332,7 +343,7 @@ rec {
 
   tezos-client = osuper.tezos-client.overrideAttrs
     (o: {
-      buildInputs = o.buildInputs ++ [ librustzcash self.makeWrapper ];
+      buildInputs = o.buildInputs ++ [ librustzcash self.makeWrapper tezos-client-alpha-commands-registration ];
```

Comment:
Wait, where is `tezos-client-alpha-commands-registration` defined?

Reply:
It's somewhere in the `ocamlPackages`, that are fulfilled by the `opam-nix` while traversing the context of the `opam-repository` :ssc:

Reply:
So you don't need to access it through `osuper` or `self.ocamlPackages.<...>` or something? :thinking: Sure would be nice to understand more about how `opam-nix` works

Reply:
there is `with oself` on the top of that file, and `self.ocamlPackages` is basically `oself`